### DISCUSSION TITLE
More options to submit id proof for student pack

### DISCUSSION THREAD

**Discussion by Prabhat1308** [community/community#8472] [Projects and Issues]
I was trying to apply for the student pack but when it turns to the id upload page it just give option for camera . No option to upload jpg.  When I tried with my phone it was still only trying to take the photo with front camera "atleast give the option for back camera that works properly" in my case it won't switch from front to back and due to bad quality of pics I got rejected 2 times . Please look into it.

**Comment by ruiming-lu** [comment]
+1 here.

**Comment by rahul0eth** [comment]
+1. Rejected 6 times. Won't use phone rear camera, so front camera is blurry, and laptop camera is blurry. Sorry Github I don't have a $1500 phone with an amazing front camera. System autorejects me everytime as "Modifications to your photo proof might have been detected by the system". Please do something GitHub.
